package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "nativescript",
   "preferGlobal": true,
-  "version": "1.0.0",
+  "version": "1.0.0-RC",
```

Comment:
RC0, RC1 maybe? What will be the next version in case we decide to publish additional one before we refactor our project infrastructure? In case we keep 1.0.0-RC here and after that we publish 1.0.0-RC2, will it work for npm?